docs/pages/modules/autolinking.mdx

```diff
@@ -0,0 +1,195 @@
+- `expo.autolinking` config object in application's **package.json**
+- per platform overrides with `expo.autolinking.ios` and `expo.autolinking.android` objects
+- options provided to the CLI command, the `use_expo_modules!` method in the **Podfile** or `useExpoModules` function in the **settings.gradle**
+
+<APIBox>
+
+### `searchPaths`
+
+A list of paths relative to the app's root directory where the autolinking script should search for Expo modules.
+It defaults to a list of all **node_modules** folders found when traversing up through your monorepo starting from the app's root directory.
+Useful when your project has a custom structure or you want to link local packages from folders different than **node_modules**.
+
+```json package.json
+{
+  "expo": {
+    "autolinking": {
+      "searchPaths": ["../../packages"]
+    }
+  }
+}
+```
+
+When used with the CLI, you can pass the search paths as command arguments like this:
+
+<Terminal cmd={['$ npx expo-modules-autolinking search ../../packages']} />
+
+</APIBox>
+<APIBox>
+
+### `exclude`
+
+A list of package names to exclude from autolinking. These packages will not be autolinked even if they are found in the search paths.
+For example, you may want not to link some packages that you don't use on the specific platform to reduce the binary size.
+The following config in **package.json** will exclude `expo-random` from autolinking on Android:
+
+```json package.json
+{
+  "expo": {
+    "autolinking": {
+      "android": {
+        "exclude": ["expo-random"]
+      }
+    }
+  }
+}
+```
+
+</APIBox>
+<APIBox platforms={['ios']}>
+
+### `flags`
+
+CocoaPods flags to pass to each autolinked pod. `inhibit_warnings` is probably the only one you may want to use to inhibit Xcode warnings produced when compiling the autolinked modules.
+You can refer to the [CocoaPods Podfile documentation](https://guides.cocoapods.org/syntax/podfile.html#pod) for available flags.
+
+<CodeBlocksTable tabs={['Podfile', 'package.json']}>
+
+```ruby
+use_expo_modules!({
+  flags: {
+    :inhibit_warnings => false
+  }
+})
+```
+
+```json
+{
+  "expo": {
+    "autolinking": {
+      "ios": {
+        "flags": {
+          "inhibit_warnings": true
+        }
+      }
+    }
+  }
+}
+```
+
+</CodeBlocksTable>
+
+</APIBox>
+
+## FAQ
+
+### How to set up the autolinking in my app?
+
+All projects created with the `npx create-expo-app` command are already configured to use Expo Autolinking. If your project was created using different tools,
+please refer to [Installing Expo modules](/bare/installing-expo-modules) to make sure your project includes all necessary changes.
+
+### What do I need to have in my module to make it autolinkable?
+
+The module resolution algorithm searches only for packages that contain the [Expo module config](./module-config.mdx) file (**expo-module.config.json**) at the root directory, next to the **package.json** file.
```

Comment:
```suggestion
The module resolution algorithm searches only for packages that contain the [Expo module config](./module-config.mdx) file (**expo-module.config.json**) at the root directory, next to the **package.json** file.
```

is `./module-config.mdx` the correct way to link to this @Simek?

Reply:
We currently have two ways of linking pages internally. I'd rather prefer it to be `/module-config.mdx` because the path resembles the actual URL. Also, using relative paths sometimes lead to "../../".